datasquare_test.go

```diff
@@ -95,47 +95,40 @@ func TestSetCell(t *testing.T) {
 	}
 }
 
+// Test_setCell verifies that setCell can overwrite cells without performing any
+// input validation.
 func Test_setCell(t *testing.T) {
```

Comment:
on re-review it seems overkill to have a test for an un-exported function that only exists for the sake of testing so I can remove this unit test if reviewers request